docs/pages/project-exporting.md

```diff
@@ -1,165 +1,129 @@
-will look in _can-tabs/src/lib/can-tabs_.
+The `system` property specifies SystemJS and StealJS overwrites. Set the [System.main] property as follows to tell SystemJS to use `src/bit-tabs.js` as the starting point of the application. The "npmIgnore" property tells StealJS to ignore processing the package.json files of certain dependencies.
 
-The "npmIgnore" property tells SystemJS to ignore processing certain dependencies's
-package.jsons.
+> In the system property, create a "main" property that points to the main entry-point. And,
+> set "npmIgnore" to ignore dependencies that aren't needed by the browser.
 
 ```
   "system": {
-    "directories" : {
-      "lib": "src"
-    },
-    "npmIgnore": ["cssify","devDependencies"]
+    "main": "src/bit-tabs",
+    "npmIgnore": ["devDependencies"]
   },
 ```
 
-> Create a "system" property that points "directories.lib" to where SystemJS should find code. And,
-> set "npmIgnore" to ignore dependencies that aren't needed by the browser.
 
-### main
+### "main" property
 
-CJS/Browserify and StealJS will read the `main` property when someone require's your package.
+CJS/Browserify and StealJS will read the `main` property when requiring your package. Set the `main` property to the CommonJS output of the export process.
 
 ```
-  "main": "lib/can-tabs",
+  "main": "dist/cjs/lib/bit-tabs",
 ```
 
-Because of the "system.directories.lib" setting, StealJS will look in _can-tabs/src/lib/can-tabs.js_, while
-Browserify will look in _can-tabs/lib/can-tabs.js_
-
-> Point "main" at the entrypoint of your package.
-
-
-### dependencies
+### "dependencies" property
 
 Use npm to install your project's dependencies.  If your project includes css or LESS files,
 include `cssify`.  Browserify will use it to bundle css files.
 
 ```
   "dependencies": {
-    "canjs": "2.2.0-alpha.8",
-    "jquery": ">1.9.0",
+    "can": "2.2.0-alpha.10",
     "cssify": "^0.6.0"
-  }
+  },
 ```
 
 Add `steal`, `steal-tools`, `grunt`, and `grunt-cli` to your project's devDependencies:
 
+> Add your project's dependencies to _package.json_.
+
 ```
   "devDependencies": {
     "grunt": "~0.4.1",
     "grunt-cli": "^0.1.13",
-    "steal": "0.5.0-pre.3",
-    "steal-tools": "0.5.0-pre.8"
+    "steal": "0.6.0-pre.0",
+    "steal-tools": "0.6.0-pre.2"
   },
 ```
 
-> Add your project's dependencies to _package.json_.
-
-### browser and browserify
+### "browser" and "browserify" properties
 
 Because our project will export CSS, we need to tell Browserify to 
 run "cssify" on css files with a `transform`.  To make this work
 with new and old versions of Browserify you must specify both the 
 "browser" and "browserify" properties.
 
+> Specify Browserify transforms.
+
 ```
   "browser": {
-    "transform": [ "cssify" ]
+    "transform": ["cssify"]
   },
   "browserify": {
     "transform": ["cssify"]
   },
 ```
 
-> Specify Browserify transforms.
+### "scripts" property
```

Comment:
can you remove all of these `"foo" property` changes? I think being nested under package.json explains that already.